frontend/src/navigations/TabBar.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import {View, Pressable, Dimensions, StyleSheet, useColorScheme} from 'react-native';
+const {width} = Dimensions.get('window');
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import {PRIMARY_COLOR} from '../Theme';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+const TabBar = ({state, descriptors, navigation}: any) => {
+    const isDarkMode = useColorScheme() === 'dark';
+  return (
+    <View style={[styles.mainContainer,{backgroundColor:isDarkMode?"black":"white"}]}>
+      {state.routes.map((route: any, index: number) => {
+        const {options} = descriptors[route.key];
+        const label =
+          options.tabBarLabel !== undefined
+            ? options.tabBarLabel
+            : options.title !== undefined
+            ? options.title
+            : route.name;
+
+        const isFocused = state.index === index;
+
+        const onPress = () => {
+          const event = navigation.emit({
+            type: 'tabPress',
+            target: route.key,
+          });
+
+          if (!isFocused && !event.defaultPrevented) {
+            navigation.navigate(route.name);
+          }
+        };
+
+        return (
+          <View
+            key={index}
+            style={[
+              styles.mainItemContainer,
+              {borderRightWidth: label == 'notes' ? 3 : 0, },
+            ]}>
+            <Pressable
+              onPress={onPress}
+              style={{
+                backgroundColor: isFocused ? PRIMARY_COLOR : isDarkMode ? "black":"white",
+                borderRadius: 50,
+                width: 60,
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}>
+              <View
+                style={{
+                  justifyContent: 'center',
+                  alignItems: 'center',
+                  flex: 1,
+                  padding: 10,
+                }}>
+                {label === 'Home' && (
+                  <Ionicons
+                    name="home"
+                    size={34}
+                    color={isFocused ? 'white' : isDarkMode?"white":"black"}
+                  />
+                )}
+                {label === 'Podcasts' && (
+                  <FontAwesome
+                    name="podcast"
+                    size={34}
+                    color={isFocused ? 'white' : isDarkMode?"white":"black"}
+                  />
+                )}
+                {label === 'Profile' && (
+                  <FontAwesome
+                    name="user-circle"
+                    size={34}
+                    color={isFocused ? 'white' : isDarkMode?"white":"black"}
+                  />
+                )}
+              </View>
+            </Pressable>
+          </View>
+        );
+      })}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  mainContainer: {
```

Comment:
Reduce Bottom to 0 or 6-7 range.

Set the margin horizontal in a way that the screen becomes within the bottom tab.

Reply:
Here is the preview bottom 7 and margin horizontal a little wider 
<img src="https://github.com/SB2318/UltimateHealth/assets/105168088/3dbf26ca-6ec4-47a2-9220-7d9eacdae340" height="600" />

Reply:
> Reduce Bottom to 0 or 6-7 range.
> 
> Set the margin horizontal in a way that the screen becomes within the bottom tab.
> 
> Reference:
> 
> ![image](https://private-user-images.githubusercontent.com/87614560/331210218-6b7803e9-0f26-4799-ae30-fd148712945a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NjQ4MzksIm5iZiI6MTcxNTg2NDUzOSwicGF0aCI6Ii84NzYxNDU2MC8zMzEyMTAyMTgtNmI3ODAzZTktMGYyNi00Nzk5LWFlMzAtZmQxNDg3MTI5NDVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDEzMDIxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRkYjc2YTk5OWJiY2Q3NzM3NWJlZDY3NGM3NDY4NWJjNDkyYWRiYzFiNGZkNDYxNTg1OTM2YmNmZWNkMjZkOTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._cz6yOWtE6ZhscI5sMHoa0n22Z3xG1BLmuZ4eQifNuk)
> 
> But don't make the icon black and white, they look nice.

Is this okay ? need changes?
<img src="https://github.com/SB2318/UltimateHealth/assets/105168088/99a98282-1c2a-48bf-be13-fc460d341368" height="600" />

Reply:
Check this one out !
<img src="https://github.com/SB2318/UltimateHealth/assets/105168088/299aa492-6730-430c-bfd3-f4f9bd27ce0c" height="600" />

Reply:
set borderBottomLeftRadius & borderBottomRightRadius to 0.

Reply:
This is the preview borderRadius 0 
<image src="https://github.com/SB2318/UltimateHealth/assets/105168088/dd538c7c-de86-4ef2-ac41-f0c2cf722b7c"  height="600"/>